### PR TITLE
testing/ostest: update mqueue testing code

### PR DESCRIPTION
**Pull request by GUIDINGLI**
## Summary
testing/ostest: update mqueue testing code
As the mqd_t change to int

### CODE REVIEW

File: testing/ostest/timedmqueue.c [hunk 1]
```diff
@@ -335,8 +344,8 @@ void timedmqueue_test(void)
   status = pthread_attr_setstacksize(&attr, STACKSIZE);
   if (status != 0)
     {
-      printf("timedmqueue_test: pthread_attr_setstacksize failed, status=%d\n",
-             status);
+      printf("timedmqueue_test: pthread_attr_setstacksize failed,"
```
Comment (xiaoxiang781216) [inline testing/ostest/timedmqueue.c:347]: add space at the end

File: testing/ostest/timedmqueue.c [hunk 2]
```diff
@@ -369,8 +378,8 @@ void timedmqueue_test(void)
...
```
Comment (xiaoxiang781216) [inline testing/ostest/timedmqueue.c:381]: add space at the end

Comment (GUIDINGLI) [reply testing/ostest/timedmqueue.c:381]: done

**Review by Ouss4 — APPROVED** [review]
LGTM

**Review by xiaoxiang781216 — APPROVED** [review]

**Pull request merged**